docs/dusd-spec.tex

```diff
@@ -472,17 +472,26 @@ \subsection{Price module}
 
 Price module:
 \begin{itemize}
-  \item The on-chain code will be written in Plutarch
-  \item The off-chain code will be built as a PAB
-  \item The datum will contain a map from timestamps (in POSIX time) to price
-    values
-  \item The script address will be dependent on three admin public
-    keys\footnote{
-      These admin keys can only be updated when the protocol is updated.
-      This decision will be enforced by using the keys to determine the script
-      address, rather than putting them into the datum.
-    }
-\end{itemize}
+  \item The off-chain code will be built in CTL.
+  \item The on-chain code will be written in Plutarch.
+  \item The validator will validate the following:
+    \begin{itemize}
+      \item There is a previous price oracle UTxO being spent.
+      \item The previous price oracle UTxO is being spent by Admin.
+      \item The previous price oracle UTxO contains the Oracle NFT.
+      \item The datum has a list of tuples containing USD Price of ADA and the POSIX timestamp 
+        of when the price was observed.
+      \item The datum list does not exceed 48 entries.
+      \item The timestamp of the latest two entries of the datum list are an hour apart.
+      \item The timestamp of the first entry in the datum list is within an hour of the timestamp
+        of the transaction.
+      \item The tail of the datum list matches the sequence of the previous price oracle UTxO's 
```

Comment:
So the list is ordered from most recent price being the first element to the furthest in the past being the last element. Does this have anyghing to do with line `490`?